assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -161,14 +205,23 @@ function Carousel() {
 
   const isAtBeginningOfList = 0 === scrollPercentage;
   const isAtEndOfList = 1 === scrollPercentage;
+  const scrollByPx = carouselSize.width;
+
+  const Item = isCompact ? CompactIndicator : DraggablePage;
+  const itemMarginRight = isCompact
+    ? CAROUSEL_COMPACT_RIGHT_MARGIN
+    : CAROUSEL_THUMB_RIGHT_MARGIN;
+  const [pageThumbWidth, pageThumbHeight] = calculatePageThumbSize(
+    carouselSize
+  );
 
   return (
     <>
       <Wrapper>
-        <Area area="left-navigation">
+        <Area area="left-navigation" marginBottom={ARROWS_BOTTOM_MARGIN}>
```

Comment:
Rather than this, create a new `NavArea` or similar, with the margin fixed:

```js
const NavArea = styled(Area)`
  margin-bottom: ${ARROWS_BOTTOM_MARGIN}px;
`;
...
<NavArea area="left-navigation">
```

Reply:
Done in 643adb5d7d5055eefe55f5a42ad8765ed0b60891